docs/quickstart.rst

```diff
@@ -279,7 +279,7 @@ This section describes how to deploy the REST Proxy container and then consume d
 
   .. sourcecode:: bash
 
-    curl -X GET -H "Accept: application/vnd.kafka.avro.v1+json" \
+    # curl -X GET -H "Accept: application/vnd.kafka.avro.v1+json" \
```

Comment:
Same explanation. Anything that runs after we've run `docker run ... bash` will give the user root shell.